app/views/handlers/thank_you.py

```diff
@@ -48,11 +48,14 @@ def get_context(self):
         if not self._is_census_theme:
             return build_default_thank_you_context(self.session_data)
 
+        schema_name = self.session_data.schema_name
         census_type_code = None
-        for census_type in self.CENSUS_TYPE_MAPPINGS:
-            if census_type in self.session_data.schema_name:
-                census_type_code = self.CENSUS_TYPE_MAPPINGS[census_type]
-                break
+
+        if "ccs" not in schema_name:
+            for census_type in self.CENSUS_TYPE_MAPPINGS:
+                if census_type in schema_name:
+                    census_type_code = self.CENSUS_TYPE_MAPPINGS[census_type]
+                    break
```

Comment:
The logic here could be kept as it was if `CENSUS_TYPE_MAPPINGS` was updated to be:

```python
    CENSUS_TYPE_MAPPINGS = {
        "census_household": "HH",
        "census_communal_establishment": "CE",
        "census_individual": "IR",
    }
```
Also, although the work was done in a previous pr, the census type code should be done in `build_census_thank_you_context`.